### DISCUSSION TITLE
28byj 48 focuser motor.

### DISCUSSION THREAD

**Discussion by Aripaule** [aaronwmorris/indi-allsky#1704] [Q&A] [answered]
HI,

![Imagem do WhatsApp de 2024-12-07 à(s) 14 42 05_7222ce02](https://github.com/user-attachments/assets/34ac3506-74c0-419d-9c7d-c295b929e5dd)

I'm using this motor for the auto focuser, and I would like to know how to move it (in and out), in the focus tab there is no option for that.

![Imagem do WhatsApp de 2024-12-07 à(s) 14 45 31_5d115f2a](https://github.com/user-attachments/assets/afc15bd0-9ba5-43a6-ba3a-346521da2457)

**Comment by aaronwmorris** [comment, marked as answer]
On the Config page, under the Devices tab, there is a config section for the Focuser (at the bottom).  Once the Focuser is configured, some new buttons will appear in the Focus view that will allow you to control the stepper motor.

https://github.com/aaronwmorris/indi-allsky/wiki/Focuser-Device

**Reply by Aripaule** [reply]
got it, thank you

**Reply by aaronwmorris** [reply]
Let me know how it works for you!

**Reply by Aripaule** [reply]
It will take a few days, I'm waiting for the power supply to arrive. (I discovered that I must power the stepper motor externally), but I'll come back here later to leave feedback. thank you.

**Reply by aaronwmorris** [reply]
I believe you can power the stepper from the 5v line of the raspberry pi.  You should at least be able to run tests without an external power source.  The current draw is really based on the amount of torque that needs to be applied, and that is not very high to just turn an all sky lens.

**Reply by Aripaule** [reply]
ok. 

I connected the step motor and driver according to the wiki page. But for some reason the module is not turning on. Focus adjustment appeared in the tools tab.


![Imagem do WhatsApp de 2024-12-07 à(s) 19 17 14_069e5fa1](https://github.com/user-attachments/assets/4581afdc-821d-4f6b-acb7-68810251508b)

**Reply by aaronwmorris** [reply]
I cannot tell based on the picture if things are wired correctly.  It looks like IN2 is not connected, though.

**Reply by Aripaule** [reply]
> I cannot tell based on the picture if things are wired correctly. It looks like IN2 is not connected, though.


It wasn't the i2, but it was something in the cables. Now the step motor worked. THANK YOU VERY MUCH!
